File: C++/1113.cpp
// Problema: 1113 - Crescente e Decrescente | Resposta: Accepted
// Linguagem: C20++ [+0s]                   | Tempo: 0.000s

#include <iostream>

int main() {
    int a, b;
    while(true){
        std::cin >> a >> b;
        if(a == b) break;
        std::cout << ((a > b) ? "Decrescente" : "Crescente") << "\n";
    }

    return 0;
}

File: JavaScript/1113.js
// Problema: 1113 - Crescente e Decrescente | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s]        | Tempo: 0.025s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var linhas = input.split('\n');
for(var linha of linhas){
    var [a, b] = linha.split(' ').map(Number);
    if(a == b) break;
    console.log((a > b) ? 'Decrescente' : 'Crescente');
}